assets/js/templates/keyboard-shortcuts-help-modal.handlebars

```diff
@@ -17,7 +17,7 @@
                 {{/if}}
               </dt>
               <dd class="shortcut-description">
-                {{this.description}}
+                {{ useTranslation this.description ../translations }}
```

Comment:
Move this to where `this.description` is defined.

Reply:
So I changed the code to use the translate function (useTranslation) in the javascript and not in the handlebars file.

assets/js/keyboard-shortcuts.js
```
import translate from './template-helpers/useTranslation'
.....
const keyboardShortcuts = [
  {
    name: 'c',
    keyCode: 67,
    description: translate('Toggle sidebar', translations),
    action: toggleSidebar
  },
```
assets/js/templates/keyboard-shortcuts-help-modal.handlebars
```
              <dd class="shortcut-description">
                {{ this.description }}
              </dd>
```